webapp/store/views.py

```diff
@@ -140,7 +140,7 @@ def details_overview(entity_name):
     channel_request = request.args.get("channel", default=None, type=str)
     package = get_package(entity_name, channel_request)
 
-    readme = package["channel_selected"]["revision"]["readme-md"]
+    readme = package["channel_selected"]["revision"].get("readme-md", "")
```

Comment:
Lets make it clear what's going on here "No readme available" or "This charm has not provided a readme.md"

Reply:
I agree, I was going to add some kind of notification. But for now I'll put `"No readme available"`

Reply:
Done.